.github/workflows/submodules.yml

```diff
@@ -0,0 +1,17 @@
+name: Update Submodules
+
+on:
+  schedule:
+    - cron: '0 21 * * *'
```

Comment:
```suggestion
    - cron: '0 21 * * *'
    - cron: '0 21 */3 * *'
```

Reply:
Revised following discussion in a3e3aa9c5156d76214e8f97e4967b9483c56a594